frontend/src/views/FieldList/FieldListModal.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
```

Comment:
This file also follows the format of AddAnimalsModal. I updated the validation behavior a bit but not much else.